.github/workflows/helm-template-generate-k8s-manifests.yaml

```diff
@@ -0,0 +1,64 @@
+name: Generate K8S Manifests from Helm Charts
+
+on:
+  # Run this when PR operations are done
+  pull_request:
+    types: [ opened, synchronize, reopened, edited ]
+
+jobs:
+  generate-k8s-manifests:
+    name: Generate K8S manifests via `helm template` command
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        kube-tag:
+          - v1.18.2
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Configure Git
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.6.2
+
+      - name: Generate manifest for cluster-prep
+        run: |
+          pushd helm/conjur-config-cluster-prep > /dev/null
+            mkdir -p generated
+
+            helm template cluster-prep . \
+              -f user-values.yaml \
+              --render-subchart-notes \
+              --skip-tests | tee generated/conjur-config-cluster-prep.yaml
+          popd > /dev/null
+
+      - name: Generate manifest for namespace-prep
+        run: |
+          pushd helm/conjur-config-namespace-prep > /dev/null
+            mkdir -p generated
+
+            helm template namespace-prep . \
+              -f user-values.yaml \
+              --render-subchart-notes \
+              --skip-tests | tee generated/conjur-config-namespace-prep.yaml
+          popd > /dev/null
+
+      - name: Commit generated manifests
+        id: commit-generated-manifests
+        run: |
+          git add helm/conjur-config-cluster-prep/generated
+          git add helm/conjur-config-namespace-prep/generated
+          git commit -m "Auto-commit of K8S manifests generated from Helm charts"
+        continue-on-error: true
```

Comment:
Scratch my comment above... If I understand correctly, the `continue-on-error: true` is needed so that the Jenkins job is not going to fail when the `git commit -m ...` fails, since this is going to be the normal, typical case where there are no changes to the generated manifests.

Reply:
Yep exactly!